addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/discovery/GlobalCacheDiscoveryService.java

```diff
@@ -165,13 +165,12 @@ private synchronized void discover() {
                     if (discoveryServiceCallback != null
                             && discoveryServiceCallback.getExistingDiscoveryResult(uid) == null
                             && discoveryServiceCallback.getExistingThing(uid) == null) {
-
                         logger.trace("Creating discovery result for: {}, type={}, IP={}", uid,
                                 gcMulticastListener.getModel(), gcMulticastListener.getIPAddress());
 
                         thingDiscovered(DiscoveryResultBuilder.create(uid)
                                 .withProperties(properties)
-                                .withLabel("GlobalCache " + gcMulticastListener.getModel())
+                                .withLabel(gcMulticastListener.getVendor() + " " + + gcMulticastListener.getModel())
```

Comment:
This line broke the build - https://github.com/openhab/openhab2-addons/pull/2172 fixes it again.
Will have to figure out why the pull request builder of Travis+Cloudbees isn't working anymore ATM :-(